src/typescript/13-utility-types/utility-types.md

```diff
@@ -247,6 +373,43 @@ const dino: SpeciesGone = "Triceratops horridus";
 
 In the code above, `SpeciesGone` is `Species` minus `Triceratops horridus`. Type `Triceratops horridus` is not assignable to the variable `dino`.
```

Comment:
This could be a little more clear that the code above will result in a TypeScript error.